round-to/roundTo.js

```diff
@@ -1,8 +1,24 @@
+const roundFunctions = {
+  up: (number, precision) => {
+    const rounder = 10 ** precision;
+    return Math.ceil(number * rounder) / rounder;
+  },
+  down: (number, precision) => {
+    const rounder = 10 ** precision;
+    return Math.floor(number * rounder) / rounder;
+  },
+};
 
-function roundTo(...args) {
-  return args;
+function roundTo(num, precision) {
+  const leastSignificantDigit = ((num * (10 ** (precision + 1))) % 10);
```

Comment:
Should simplify this step by using good variable names, so that the logic will be clear an easy to understand.